### ISSUE TITLE
Online Editor - Default text in comment field not showing in running survey

### ISSUE THREAD

**Issue by steff381** [surveyjs/editor#169]
### Are you requesting a feature, reporting a bug or ask a queation?
Bug

### What is the current behavior?
In a comment field, if there is a default answer the default text does not display in the running survey nor does it appear in the results. 

### What is the expected behavior?
Default text from comment field is displayed in running survey and is output into the results

### How would you reproduce the current behavior (if this is a bug)?
Create a comment field in survey editor. 
Create a default value in property edit window
Run a survey (default text not displayed)

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
your_code_here

```

### Specify your

* browser: Chrome
* editor version: 0.12.31

**Comment by andrewtelnov** [comment]
@steff381 Right, I did not think about question comments. We will fix it.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@steff381 Or do you speak about QuestionComment? Because it works fine for me...

Thank you,
Andrew

**Comment by steff381** [comment]
The comments are correctly being injected into the JSON in the "test survey" tab of the Editor page on surveyjs.io. When I tested a second time I noticed that the issue seems to be related to the dxsurvey.com implementation of the surveyjs service. If I run the survey using the "published" endpoint the default value is not present and the content does not make it to the final output. Does that make sense?
